rtos/TARGET_CORTEX/mbed_boot.h

```diff
@@ -0,0 +1,159 @@
+/* mbed Microcontroller Library
```

Comment:
You are using doxy, but it's not going to be anchored nicely. Is it worth creating a group and turning all the comments (even the file global one) into doxy?